cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -122,6 +129,20 @@ permissions:
 
   cluster_status_get:
     - sys_admin
+  cluster_status_put:
```

Comment:
why do we need this?
we only need the next three

Reply:
Inbal and I thought about it, and we said that maybe it would be better to leave it so we can have a "primary user" to use for authorization of the `put` function in the [ClusterStatus class](https://github.com/cloudify-cosmo/cloudify-manager/blob/73aee84dee538850f766edd97272f7f56169081d/rest-service/manager_rest/rest/resources_v3_1/cluster_status.py#L69). Otherwise, as I see it now, it would be left only with the authorization of the inheriting `put` functions.